modules/ROOT/pages/first-successful.adoc

```diff
@@ -4,53 +4,64 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: routers, flows, processor chain, async, aggregator, resequencer
 
-The First Successful router iterates through its list of child routes, each containing a set of components, until the first successful execution. If none succeed, an exception is thrown.
+The First Successful router iterates through a list of configured processing routes until one of them executes successfully. If any processing route fails its execution (throws an exception), the router executes the next configured route.
```

Comment:
Very minor suggestions: 

 "...one of the routes executes successfully."
"...fails execution (throws and exception)..."